services/movie_session.py

```
@@ -16,6 +16,7 @@ def create_movie_session(
 def get_movies_sessions(session_date: str = None) -> QuerySet:
     queryset = MovieSession.objects.all()
     if session_date:
+        session_date = datetime.fromisoformat(session_date).date()
```

Parsing session_date with `datetime.fromisoformat(session_date).date()` is fine, but be explicit when checking optional args. Prefer `if session_date is not None:` to avoid silently skipping empty strings. This mirrors the checklist guidance that used `is not None` for optional filters (e.g., for title filtering in other services).